go.mod

```diff
@@ -32,21 +32,21 @@ require (
 	go.etcd.io/etcd/client/pkg/v3 v3.5.4
 	go.etcd.io/etcd/server/v3 v3.5.0
 	go.uber.org/multierr v1.7.0
-	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
+	golang.org/x/net v0.5.0
 	google.golang.org/genproto v0.0.0-20220519153652-3a47de7e79bd
 	gopkg.in/square/go-jose.v2 v2.2.2
-	k8s.io/api v0.24.3
+	k8s.io/api v0.26.1
```

Comment:
Wouldn't have expected all these bumps?

Reply:
huh. I just attempted to bump the client-go bits. sec.

Reply:
Oops, did a `go get -u ...` ... fixed!